### DISCUSSION TITLE
[OKD 4.15] Disable Auto Cluster Update

### DISCUSSION THREAD

**Discussion by parseltongued** [okd-project/okd#1981] [General]
**Describe the bug**
I am using OKD in an air-gap cluster. Currently my 3-masters are tainted because cluster update failing. All pods are scheduling to the master-1.
![image](https://github.com/user-attachments/assets/1596b4f4-7fbf-47a0-a80a-5af39c74c01d)

Running the command below was successful but doesn't have any change to the taint.
```
kubectl taint nodes control-plane0.okd4.<domain> UpdateInProgress-
node/control-plane0.okd4.<domain> untainted
```
I've also paused the update on Admin Operator UI but no effect:
![image](https://github.com/user-attachments/assets/fff9d708-87a5-49e0-b03b-c034c62b9ba7)

How to disable cluster update and more importantly remove the taint.

**Version**
Deployment information
OKD Cluster Version: 4.15.0-0.okd-2024-03-10-010116
Kernel version: v1.28.2-3598+6e2789bbd58938-dirty
Installation method: Bare-metal UPI (Airgapped, self hosted quay)

Thank you

**Comment by melledouwsma** [comment]
Personally, I would address the issues causing the update to fail, then allow the update to complete to restore the cluster to its normal state. Could you provide more details on why the update is failing?

Pausing the MachineConfigPools for longer periods of time has some consequences, you can find out more about that on [The Consequences Of Pausing MachineConfig Pools In OpenShift's Machine Config Operator](https://www.redhat.com/en/blog/the-consequences-of-pausing-machineconfig-pools-in-openshifts-machine-config-operator-mco).
